<ui version="4.0" >
 <class>dlgAbout</class>
 <widget class="QDialog" name="dlgAbout" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>About</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>258</y>
     <width>500</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblDescription" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>480</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCopyright" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>60</y>
     <width>480</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVersion" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>100</y>
     <width>480</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCLIReference" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>140</y>
     <width>480</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLotROLinuxReference" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>200</y>
     <width>480</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>dlgAbout</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>259</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel" >
     <x>259</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
